packages/cli/src/NodeTypes.ts

```diff
@@ -16,12 +16,8 @@ import type { Dirent } from 'fs';
 
 @Service()
 export class NodeTypes implements INodeTypes {
-	constructor(private nodesAndCredentials: LoadNodesAndCredentials) {}
-
-	init() {
-		// Some nodeTypes need to get special parameters applied like the
-		// polling nodes the polling times
-		this.applySpecialNodeParameters();
+	constructor(private nodesAndCredentials: LoadNodesAndCredentials) {
```

Comment:
Nit: Sometimes we rename this dependency, other times we use the full name. Let's keep it consistent.